ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+attempts = 0
+
+until attempts == 4
+	attempts += 1
+
+puts "What is your name?"
+name = gets.chomp
+	
+
+def age_finder(x, y)
+		 (2016 - y.to_i) == x.to_i
+end
+
+puts "How old are you? What year were you born?"
+age = gets.chomp
+year = gets.chomp
+
+	
+puts "The cafeteria serves garlic bread. Should we order some for you?"
+garlic_bread = gets.chomp
+
+	if garlic_bread == "yes"
+	  garlic_bread = true
+	else
+		garlic_bread == "no"
+	  garlic_bread = false
+	end
+
+puts "Would you like to enroll in the company's health insurance?"
+insurance = gets.chomp
+
+	if insurance == "yes"
+	  insurance = true
+	elsif insurance == "no"
+	  insurance = false
+	else insurance == nil
+		insurance = false
+	end
+
+
+if age_finder(age.to_i, year.to_i) && (garlic_bread || insurance)
+	puts "Probably not a vampire"
+elsif (age_finder(age.to_i, year.to_i)) &&  (garlic_bread == "no") && (insurance ==nil)
```

Comment:
Above you set garlic_bread and insurance to boolean values, make sure you use those in the conditional flow and statements! You can use the boolean variable for `garlic_bread` in your conditionals like:
```
if age! && (garlic_bread || health_insurance)
```
would be checking to see if the employee lied about their age and either they wanted garlic bread or wanted health insurance. You can also use the `!` operator to check for a false variable like:
```
elsif finalAge! && (garlic_bread! || health_insurance!)
``` 
will check to see if the age was false and either the garlic bread or health insurance was set to false.